docs/sources/auth/generic-oauth.md

```diff
@@ -183,6 +183,7 @@ allowed_organizations =
     scopes = openid email name
     auth_url = https://login.microsoftonline.com/<directory id>/oauth2/authorize
     token_url = https://login.microsoftonline.com/<directory id>/oauth2/token
+    signout_redirect_url = https://login.microsoftonline.com/<directory id>/oauth2/logout?post_logout_redirect_uri=https://<grafana domain>
```

Comment:
`signout_redirect_url` is not supported per OAuth provider, only on a global level in the `[auth]` section and is already documented [here](https://grafana.com/docs/grafana/latest/auth/overview/#url-redirect-after-signing-out).